repository/v.json

```diff
@@ -393,6 +393,18 @@
 				}
 			]
 		},
+		{
+			"name": "Vintage Relative Lines",
+			"details": "https://github.com/bicarlsen/vintage_relnums",
+			"issues": "https://github.com/bicarlsen/vintage_relnums/issues",
+			"labels": [ "vintage", "line numbers" ],
+			"releases": [
+				{
+					"sublime_text": "*",
```

Comment:
`"> 3000"` would probably more correct here?

Reply:
Yes, thank you. I hadn't read through the ST2 Docs yet, but it appears it isn't compatible with it. I'll make the change.